rerun_cpp/src/rerun/as_components.hpp

```diff
@@ -47,6 +47,16 @@ namespace rerun {
         }
     };
 
+    /// AsComponents for std::initializer_list
+    template <typename TComponent>
+    struct AsComponents<std::initializer_list<TComponent>> {
```

Comment:
great addition! All others have a unit test in `tests/recording_stream.cpp`, but it's trivial so we can let it slide